wg-manifests/charter.md

```diff
@@ -0,0 +1,100 @@
+# WG Manifests Charter
+
+This charter adheres to the conventions, roles and organization management
+outlined in [wg-governance].
+
+## Scope
+
+- Provide a catalog (centralized repository) of Kubeflow application manifests.
+- Provide a catalog of third-party apps for common services.
+- Provide documentation to help users install a set, or all of the included
```

Comment:
> Provide documentation to help users install a set, or all of the included
  apps.

Please remove this from the scope of the charter.

The primary goal with forming WGs right now is to reduce ambiguity by clarifying ownership of existing assets. 

This  is not an existing asset.

Furthermore, my expectation is that application owners will be responsible for instructions to install their application standalone.

Distribution owners will be responsible for providing instructions to install their distributions.

Can we see how that goes before expanding the scope of this WG? If we find that leaves a gap and this WG ends up addressing the gap then we can expand the scope later on.